packit/distgit.py

```diff
@@ -316,11 +316,13 @@ def download_upstream_archive(self) -> Path:
         with cwd(self.local_project.working_dir):
             self.download_remote_sources(self.config.pkg_tool)
         archive = self.absolute_source_dir / self.upstream_archive_name
+        logger.info(f"Downloading archive: {self.upstream_archive_name}")
```

Comment:
I think it should go [here](https://github.com/packit/packit/blob/902c61c2f2ef41873009755342a0e535bcb8b7b4/packit/base_git.py#L493). At its current place it will be printed »after« downloading and it doesn't respect multiple sources.

Reply:
In my case, the source is not present in `sources` file. So the `download_remote_soureces()` will simply not download the file - without exception.  It will not even reach the section you linked. Just the required archive will not exist.
But I agree, that sometimes, you may get TB and not printed this info().

I updated PR.